restcomm/restcomm.mscontrol.jsr309/src/main/java/org/restcomm/connect/mscontrol/jsr309/Jsr309BridgeController.java

```diff
@@ -264,15 +263,25 @@ private void saveRecording() {
                 builder.setAccountSid(accountId);
                 builder.setCallSid(callId);
                 builder.setDuration(duration);
-                builder.setApiVersion(runtimeSettings.getString("api-version"));
+                String apiVersion = runtimeSettings.getString("api-version");
```

Comment:
shall we replace hardcoded "api-version" with String constant that can be reused?